floodrelief/settings.py

```diff
@@ -40,8 +40,8 @@ def get_list(text):
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
 
-# ALLOWED_HOSTS = ['127.0.0.1', 'keralarescue.herokuapp.com', 'keralarescue.in', 'www.keralarescue.in', 'localhost']
-ALLOWED_HOSTS = get_list(os.environ.get('ALLOWED_HOSTS'))
+ALLOWED_HOSTS = ['127.0.0.1', 'keralarescue.herokuapp.com', 'keralarescue.in', 'www.keralarescue.in', 'localhost']
+#ALLOWED_HOSTS = get_list(os.environ.get('ALLOWED_HOSTS'))
```

Comment:
Revert the changes done to this file.

Reply:
line 44 should be as it is. we should use envs!